doc/intel-gpu-dkms.rst

```diff
@@ -81,7 +81,7 @@ Install Intel GPU Firmware and DKMS packages::
   sudo cp /opt/packages/firmware/*.bin /lib/firmware/updates/i915/
   sudo dpkg -i /opt/packages/intel-i915-dkms_*.deb \
     /opt/packages/intel-platform-cse-dkms-*.deb \
-    /opt/packages/intel-platform-pmt-dkms-*.deb
+    /opt/packages/intel-platform-vsec-dkms-*.deb
```

Comment:
Did you have a chance to check how installing vsec.deb plays out on top of pmt.deb? Any issues observed worth to be mentioned in troubleshoot section?

Reply:
From what I tested, the deb package created starting from 449 are named as vsec.deb instead of pmt.deb. Should be just naming change